### DISCUSSION TITLE
Flash does not work

### DISCUSSION THREAD

**Discussion by Th0tti** [jomjol/AI-on-the-edge-device#1383] [General]
With my esp32 cam, flashing shows that everything has been installed.  Then I get the following error and the device is in a boot loop.

 E (883) psram: PSRAM ID read error: 0xffffffff
 E (883) cpu_start: Failed to init external RAM!

**Comment by caco3** [comment]
That indicates that your hardware is bad!
Either the external PSRAM is broken or it has not the required  4MB RAM
